docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -5,4 +5,26 @@ This guide contains instructions to upgrade from version v7.1.0 to Unreleased.
 **Before you start, don't forget to take a look at [general instructions](/UPGRADE.md) about upgrading.**
 There you can find links to upgrade notes for other versions too.
 
+## [shopsys/project-base]
+### Configuration
+- translations are now extracted also from overwritten templates in `app/Resources` ([#931](https://github.com/shopsys/shopsys/pull/931))
+    - update your `build-dev.xml` file accordingly:
+        ```diff
+            <target name="dump-translations-project-base" description="Extracts translatable messages from all source files in project base.">
```

Comment:
what about `dump-translations-project-base` target in `build.xml` of monorepo root ?